tests/unit/test_noise_form.py

```diff
@@ -252,7 +254,7 @@ def test_columns_noised(dummy_data: pd.DataFrame) -> None:
     ],
 )
 def test_correct_datasets_are_used(
-    func: Callable, dataset_schema: DatasetSchema, mocker: MockerFixture
+    func: Callable[..., pd.DataFrame], dataset_schema: DatasetSchema, mocker: MockerFixture
```

Comment:
And here

Reply:
Same issue as with DATASET_GENERATION_FUNCS